src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableDataSource.cs

```diff
@@ -20,12 +20,25 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
 using Microsoft.VisualStudio.Shell.TableManager;
 using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Core.Helpers;
+using SonarLint.VisualStudio.IssueVisualization.Editor.LocationTagging;
+using SonarLint.VisualStudio.IssueVisualization.Models;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
 {
-    internal sealed class HotspotsTableDataSource : ITableDataSource, IDisposable
+    internal interface IHotspotsStore
+    {
+        void Add(IAnalysisIssueVisualization hotspot);
```

Comment:
This is the method that the HttpRequest handler can call.

Reply:
Yep - except it won't be the HTTP request handler itself; it'll be a class that knows nothing about HTTP.